core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -8,25 +13,278 @@ p-table {
+            & .p-sortable-column-icon {
+                display: inline-block;
+                color: $color-palette-gray-700;
+                margin-left: $spacing-1;
+            }
+
+            & .p-sortable-column-badge {
+                border-radius: 50%;
+                height: 1.143rem;
+                min-width: 1.143rem;
+                line-height: 1.143rem;
+                color: $color-palette-primary;
+                background: $bg-highlight;
+                margin-left: $spacing-1;
+            }
+
+            &:not(.p-highlight):hover {
+                color: $text-color-hover;
+
+                & .p-sortable-column-icon {
+                    color: $black;
+                }
+            }
+
+            &.p-highlight {
+                & .p-sortable-column-icon {
+                    color: $color-palette-primary-500;
+                }
+            }
+        }
+
+        & .p-datatable-tbody > tr {
+            background: $white;
+            color: $black;
+            height: auto;
+            transition: none;
+            outline: 0 none;
+            cursor: pointer;
+            overflow-wrap: anywhere;
+
+            &:hover {
                 background-color: $color-palette-primary-200;
             }
+
+            & > td {
+                text-align: left;
+                border: 1px solid $color-palette-gray-300;
+                border-width: 0 0 1px 0;
+                padding: 0.625rem $spacing-1;
+                font-size: $font-size-smd;
+
+                &:first-child {
+                    padding-left: $spacing-3;
+                }
+
+                &:last-child {
+                    padding-right: $spacing-3;
+                }
+
+                & .p-row-toggler,
+                & .p-row-editor-init,
+                & .p-row-editor-save,
+                & .p-row-editor-cancel {
+                    width: $spacing-5;
+                    height: $spacing-5;
+                    color: $black;
+                    border: 0 none;
+                    background: transparent;
+                    border-radius: 50%;
+                    transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
+
+                    &:enabled:hover {
+                        color: $text-color-hover;
+                        border-color: transparent;
+                        background: $color-palette-primary-100;
+                    }
+
+                    &:focus {
+                        outline: 0 none;
+                        outline-offset: 0;
+                        box-shadow: none;
+                    }
+                }
+
+                & .p-row-editor-save {
+                    margin-right: $spacing-1;
+                }
+
+                .pi.pi-ellipsis-v {
+                    color: $color-palette-primary-500;
+                }
+            }
+
+            &.p-highlight {
+                background: $bg-highlight;
+                color: $text-color-highlight;
```

Comment:
does the highlight state of a table gets shared with other highlights in the design system? if so we need these vars.

Reply:
will use direct variable instead, reverting this change: https://github.com/dotCMS/core/pull/28359/files#diff-2728d896cd6de95b52033ed77ebb0e002a807f786c31802032738931ea1c462bL24